sandbox/modules/identity/src/main/java/org/opensearch/identity/support/ConfigHelper.java

```diff
@@ -162,4 +165,14 @@ public static <T> SecurityDynamicConfiguration<T> fromYamlString(
         return fromYamlReader(new StringReader(yamlString), ctype, version, seqNo, primaryTerm);
     }
 
+    public static String generateCommonLangPassword() {
```

Comment:
Relevant: https://xkcd.com/936/

I would prefer easier to copy-paste vs 'complex characters'

Reply:
Note; this was a nit, feel free to keep as is or change it up.